src/backends/fvm.hpp

```diff
@@ -2,6 +2,6 @@
 
 #include "fvm_multicore.hpp"
 
-#ifdef WITH_CUDA
+#ifdef NMC_HAVE_CUDA
```

Comment:
WITH for user options, HAVE in the code. not very difficult to remember, but admittedly a pita for you to change.